src/types.ts

```diff
@@ -33,6 +34,11 @@ export interface SyncOptions {
    * syncing jobs would overwrite each other.
    */
   uniqueId?: string
+  /**
+   * When enabled, the `stats` event will be emitted after each batch is processed.
+   * Stats will be zero if the corresponding option is not set - `maxItemsPerSec` or `maxBytesPerSec`.
```

Comment:
This behavior seems a little unexpected. Would it be easy to make it so that stats are always available, even if there is no throttling?

Reply:
Agreed. I removed the option in favor of always emitting the `stats` event. It's safe to do that since it will be zero if the corresponding option is not set.